packages/commercetools/composables/src/types/Locale.ts

```diff
@@ -0,0 +1,9 @@
+import { Ref } from '@vue/composition-api';
+import { LocaleItem } from '@vue-storefront/commercetools-api/lib/src/types/setup';
+
+export type Locale = Ref<string>
+export type Country = Ref<string>
+export type Currency = Ref<string>
+export type AvailableLocales = Ref<Readonly<LocaleItem[]>>
```

Comment:
If we apply the above we can also get rid of this type (and `AvailableCurrencies`, `AvailableCountries`